### ISSUE TITLE
Svg rendering regression (0.11.0+1 to 0.12.0+)

### ISSUE THREAD

**Issue by MisterJimson** [dnfield/flutter_svg#170]
I have an SVG that renders as expected on 0.11.0+1 and below, but has a missing part on 0.12.0 and above.

Source SVG: https://drive.google.com/file/d/1UmyZ-qhOF_lD68PhIXDRf9gFVKBdyY7Y/view?usp=sharing

0.11.0+1 how it looks: https://i.imgur.com/Wtx7YRT.png
0.12.0 and above how it looks: https://i.imgur.com/gH52yTa.png

**Comment by dnfield** [comment]
Hmm.  Problem seems somehow related to opacity processing..  checking.

**Comment by dnfield** [comment]
Really strange.  As a work around, if you move the `opacity` from the group down to the paths it works:

```svg
<svg viewBox="0 0 876.83 232.87"
  xmlns="http://www.w3.org/2000/svg">
  <g fill="#fff">
    <g transform="translate(-72.33 -72.55)">
      <path d="m452 138.41v101h-20v-34.21h-46.61v34.25h-19.92v-101h19.92v49.16h46.61v-49.2z"/>
      <path d="m580 189c0 28.52-23.93 51.59-53.74 51.59s-53.77-23.06-53.77-51.59 24.07-51.6 53.74-51.6 53.77 23.08 53.77 51.6zm-20.92 0c0-17.78-14.76-32.39-32.82-32.39a32.39 32.39 0 1 0 32.79 32.39z"/>
      <path d="m704.47 138.41v101h-20.64v-55.85l-31.24 43.85-31.39-43.85v55.89h-20.49v-101h10l41.85 57.9 41.85-57.9z"/>
      <path d="m809.48 221.68v17.77h-77.54v-101h74.53v17.78h-54.61v25.65h45.72v17.12h-45.72v22.64z"/>
      <path d="m925.37 138.41h22.79l-34.4 50 35.4 51h-23.36l-25.37-35.21-25.37 35.25h-23.36l35.55-51-34.54-50h22.93l24.79 34.4z"/>
    </g>
    <path d="m827.48 180.1h21.89v21.89h-21.89z" transform="matrix(.70710678 -.70710678 .70710678 .70710678 38.15 576.27)"/>
    <g transform="translate(-72.33 -72.55)">
      <path opacity=".65" d="m72.33 257.66v47.76h56a24.3 24.3 0 0 0 24.18-24.42v-45.36l133-73.75.32-.17v-22.47zm61.35-12.18v32.91c0 4.11-3.49 6.33-6.77 6.33h-34.13v-16.56z"/>
      <path opacity=".65" d="m285.82 120.3v-47.75h-56a24.3 24.3 0 0 0 -24.16 24.38v45.39l-133 73.75-.32.18v22.46zm-60.55 10.08v-32.91c0-4.11 3.49-6.33 6.77-6.33h34.13v16.56z"/>
    </g>
    <path d="m267 268.16v16.56h-34.16c-3.28 0-6.77-2.22-6.77-6.33v-32.91zm-194.67-128.91v22.47l.32.17 133 73.75v45.36a24.3 24.3 0 0 0 24.16 24.39h56v-47.73z" opacity=".85" transform="translate(-72.33 -72.55)"/>
    <path d="m92.78 107.7v-16.56h34.13c3.28 0 6.77 2.22 6.77 6.33v32.91zm193 131v-22.46l-.32-.18-133-73.75v-45.38a24.3 24.3 0 0 0 -24.16-24.38h-56v47.75z" opacity=".85" transform="translate(-72.33 -72.55)"/>
  </g>
</svg>
```

For some reason, it also seems to work if you remove the other path that looks good.  

Still looking into why.

**Comment by dnfield** [comment]
(FWIW, not using group opacity should also perform slightly better since it doesn't need to do a saveLayer)

**Comment by dnfield** [comment]
Ok.  That's the problem.  The save layer is getting the wrong bounds. Or rather, we're not transforming the bounds correctly.  

The call to `saveLayer` takes a Rect. We're passing through the bounding rect without transforming it - but your group has a transform on it, so we end up clipping.  I can fix this, but temporarily that work around should fix it (because it no longer requires the save layer).

**Comment by dnfield** [comment]
This is fixed on 0.12.4+2 for flutter <= 1.5.8, and on 0.13.0+2 for flutter > 1.5.8
